cardano-cli/src/Cardano/CLI/EraBased/Query/Run.hs

```diff
@@ -1101,11 +1102,22 @@ writeStakeAddressInfo
         )
         merged
 
-    friendlyDRep :: L.DRep -> Text
+    friendlyStake :: PoolId -> Aeson.Value
+    friendlyStake poolId@(StakePoolKeyHash (C.KeyHash hash)) =
+      Aeson.object
+        [ "stakePoolBech32" .= serialiseToBech32 poolId
+        , "stakePoolHex" .= hashToTextAsHex hash
```

Comment:
I think this may be more readable
```suggestion
        [ "stakePoolBech32" .= UsingBech32 poolId
        , "stakePoolHex" .= UsingRawBytesHex hash
```